packages/aws-rfdk/lib/deadline/lib/repository.ts

```diff
@@ -387,6 +421,15 @@ export interface RepositoryProps {
    * @default Repository settings are not imported.
    */
   readonly repositorySettings?: Asset;
+
+  /**
+   * Define the settings used by Deadline Secrets Management, a feature introduced in Deadline 10.1.10 for securely managing storage
+   * and access of Secrets for your render farm.
+   * More details at:
+   * https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/secrets-management/deadline-secrets-management.html
+   * @default: Secrets Management will be enabled and a username and password will be automatically generated if none are supplied.
+   */
+  readonly secretsManagementSettings?: SecretsManagementProps
```

Comment:
```suggestion
  readonly secretsManagementSettings?: SecretsManagementProps;
```
Does our linting during the build not catch that? Seems like it should

Reply:
Also missing on the class variable on line 555.